docs/self-managed/backup-restore/operate-tasklist-backup.md

```diff
@@ -5,6 +5,12 @@ description: "How to perform a backup and restore of Operate and Tasklist data."
 keywords: ["backup", "backups"]
 ---
 
+:::note
+This release introduces breaking changes, including the [get backup state API and response codes](#get-backup-state-api), [get backups list API](#get-backups-list-api), and notably the utilized URL.
+
+For example, `curl 'http://localhost:8080/actuator/backups'` rather than the previously used `backup`.
```

Comment:
@sdorokhova Can we make a note here on backwards compatibility or is this still subject to change?

Reply:
@sdorokhova is on vacation -- I'll launch an additional PR depending on feedback upon return.